Doc/library/urllib.error.rst

```diff
@@ -48,14 +53,20 @@ The following exceptions are raised by :mod:`urllib.error` as appropriate:
    .. attribute:: reason
 
       This is usually a string explaining the reason for this error.
+      Is an alias for *msg* attribute.
 
    .. attribute:: headers
 
       The HTTP response headers for the HTTP request that caused the
       :exc:`HTTPError`.
+      Is an alias for *hdrs* attribute.
 
       .. versionadded:: 3.4
 
+   .. attribute:: fp
+
+      A file-like object with the HTTP error body.
```

Comment:
The file like object does not have the body, you can read from it. I'd write something like "A file-like object where the HTTP error body can be read from."